book/src/caveat.md

```diff
@@ -1,5 +0,0 @@
-> ⚠️ **IN-PROGRESS VERSION OF SALSA.** ⚠️
->
-> This page describes the unreleased "Salsa 2022" version, which is a major departure from older versions of salsa. The code here works but is only available on github and from the `salsa-2022` crate.
->
-> If you are looking for the older version of salsa, simply visit [this link](https://salsa-rs.netlify.app/salsa)
```

Comment:
We probably want some *variant* of this to indicate that it's not what is on crates.io

Reply:
Filed https://github.com/salsa-rs/salsa/issues/503